client/app/telecom/telephony/alias/administration/terminate/telecom-telephony-alias-administration-terminate.controller.js

```diff
@@ -81,5 +81,48 @@ angular.module("managerApp").controller("TelecomTelephonyAliasAdministrationTerm
         });
     };
 
+    self.bulkDatas = {
+        billingAccount: $stateParams.billingAccount,
+        serviceName: $stateParams.serviceName,
```

Comment:
You can purge bulkDatas of these two params. You could use number.billingAccount / serviceName of self.number in the view (see comment below)